lib/oauth/apple/idtoken_test.go

```diff
@@ -0,0 +1,291 @@
+package apple
```

Comment:
should this be `package apple_test`?

Reply:
Its testing unexported functions that I'm not sure we need to export - but I did move the mock keyStore into the  `_test` file.

Moving to `apple/test` probably works as well.

Reply:
Actually this got a lot more practical when I implemented that claims suggestion.